src/appleseed-max-impl/appleseedrenderer/appleseedrenderer.cpp

```diff
@@ -1377,5 +1377,8 @@ const MCHAR* AppleseedRendererClassDesc::GetRsrcString(INT_PTR id)
 {
     const auto* dialog_string_pair = LOOKUP_KVPAIR_ARRAY(g_dialog_strings, id);
 
-    return dialog_string_pair->m_value;
+    if (dialog_string_pair != nullptr)
```

Comment:
In what cases would we end up with a null pointer here?

Reply:
In cases of paramblock parameters when they don't have resource string ID.
In renderer pblock all the parameters have 0 for string id.

Reply:
I couldn't find any use of sring resources for pblock params, so I decided to put 0 and it looks fine. Less resource entries looked good thing to me.